### ISSUE TITLE
Record drop from the app and upload directly to Soundcloud

### ISSUE THREAD

**Issue by carpodaster** [sounddrop/sounddrop#86]
- [ ]  Capture audio via [getUserMedia()](http://www.html5rocks.com/en/tutorials/getusermedia/intro/#toc-round3)
- [ ] Capture lat/lng using the Geolocation API
- [ ] Upload Drop to Soundcloud

**Comment by langjoh** [comment]
@carpodaster So users wouldn't need an extra recording app on their phone, but instead they can use SoundDrop? Just to understand what this issue means...

**Comment by carpodaster** [comment]
Yes, a native mobile app would have access to the microphone (provided the user grants access to it). But even w/o a native mobile app (ie. one that you download and install from an app store), modern HTML5 provides the required APIs as well. Again, provided the user grants microphone access to the web page.

**Comment by berlintam** [comment]
I think one can record a sound directly to soundcloud using the soundcloud app... if I'm not mistaken..... So I guess one could in theory do it the other way around... upload a sound to soundcloud & then link it up w/ sounddrop.

**Comment by carpodaster** [comment]
Yes, that works of course. You'd have to switch back and forth between apps, though. As a user, I would prefer doing everything without "alt-tabbing".

**Comment by berlintam** [comment]
I just stumbled upon this: https://www.codecademy.com/courses/javascript-intermediate-en-txGOj/0/1

There's a section in here that covers recording a sound using the api! Exactly what we want! (That being said, I haven't looked at the exercise at all, i've just read the table of contents)
